keras/backend/tensorflow_backend.py

```diff
@@ -1828,6 +1828,25 @@ def less_equal(x, y):
     return tf.less_equal(x, y)
 
 
+def where(condition, x, y):
+    """Returns elements, either from x or y, depending on condition.
+       If only condition is given, (suppose we have transfer condition from Tensor to `numpy.ndarray`)
+       Return `tf.convert_to_tensor(condition.nonzero())`.
+
+    # Arguments
+        condition: Tensor or variable.
+        x: Tensor or variable.
+        y: Tensor or variable.
+
+    # Returns
+        A tensor.
+    """
+    if x is not None and y is not None:
```

Comment:
It seems like this `if` statement would have no effect?

Reply:
Thx, I had it revised.